kbchat/kbchat_test.go

```diff
@@ -120,15 +135,26 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Error in preparing working directory: %v\n", err)
 	}
-	kbc, err = Start(RunOptions{KeybaseLocation: kbLocation, HomeDir: dir, Oneshot: config.Bots.Alice1, StartService: true})
+	alice, err = Start(RunOptions{KeybaseLocation: kbLocation, HomeDir: dir, Oneshot: config.Bots.Alice1, StartService: true})
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Error in starting service: %v\n", err)
 	}
 
+	channel = Channel{
+		Name: fmt.Sprintf("%s,%s", config.Bots.Alice1.Username, config.Bots.Charlie1.Username),
+	}
+	teamChannel = Channel{
+		Name:        config.Teams.Acme.Teamname,
+		Public:      false,
+		MembersType: "team",
+		TopicName:   config.Teams.Acme.Channel,
+		TopicType:   "chat",
+	}
```

Comment:
these aren't really testing anything in here. it looks like they're being set for later, but that's polluting the outer namespace. we can do this more cleanly.